docs/getting-started/smi/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: rollouts-demo-stable
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/configuration-snippet: |
```

Comment:
What do you think about adding a comment that links to the linkerd docs explaining this line?

Reply:
good idae